compiler/loco/include/loco/IR/Graph.h

```diff
@@ -264,8 +264,7 @@ struct GraphOutputIndexQueryService : public DialectService
   virtual GraphOutputIndex index(const Node *node) const = 0;
 };
 
-// TODO Use "const Graph *"
```

Comment:
Hmm.. may be better to simply remove this TODoO.. ^^;

Ref: https://mlir.llvm.org/docs/Rationale/UsageOfConst/